index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+      <header id="header">
+        <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"></img>
+
+        <nav id="nav-bar">
+          <ul>
+            <li><a class="nav-link" href="#section1">Features</a></li>
+            <li><a class="nav-link"  href="#section2">How It Works</a></li>
+            <li><a class="nav-link"  href="#section3">Pricing</a></li>
+          </ul>
+        </nav>
+      </header>
+
+
+
+      <section id="hero">
```

Comment:
Don't leave blank spaces, Your code could be clean if you avoid making newlines or blank spaces between
</header> and the section.

It will be better if you make the change 😃